recipes/pyodps/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "pyodps" %}
+{% set version = "0.12.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/pyodps-{{ version }}.tar.gz
+  sha256: e1820a1fa152f5d6f07572adadc1a0271431f754da2d710a1b2678daa9ec2e6a
+
+build:
+  entry_points:
+    - pyou = odps_scripts.pyou:main
+    - pyodps-pack = odps_scripts.pyodps_pack:main
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ stdlib('c') }}
+  host:
+    - python
+    - pip
+    - setuptools <64  # [py<=311]
+    - setuptools <70  # [py>311]
+    - wheel
+    - numpy >=2.0.0
+    - cython >=3.0,<3.1  # [not win]
+  run:
+    - python
+    - requests >=2.4.0
+    - pyarrow >=2.0.0
+    - {{ pin_compatible('numpy') }}
+
+test:
+  imports:
+    # cupid appears to be imported as part of the pyodps[mars] 
+    # extra that we're not planning to support
+    # - cupid
+    - odps
+    - odps_scripts
```

Comment:
It appears that `pyodps` installs these three python packages.  It doesn't look like `cupid` is a vendoring of anything else.  But it does look like a kind of a sloppy packaging process.  It doesn't look like `cupid` is used except by the `mars` extra, which I'm not planning to support on conda-forge for now because its `pymars` dependency isn't available here.